osmosis/assetlist.json

```diff
@@ -23842,6 +23842,64 @@
       "name": "Mars Looped USDC",
       "display": "loopedUSDCmars",
       "symbol": "loopedUSDCmars"
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "factory/osmo1xu0gk9aggv79597xwazyfzaggv2pze9z7cq3p9p72tkkux9a7xaqufa792/BVT",
+          "exponent": 0
+        },
+        {
+          "denom": "BVT0",
+          "exponent": 18
+        }
+      ],
+      "description": "Banana Vault Token - Peelworks Factory II",
+      "type_asset": "sdk.coin",
+      "base": "factory/osmo1xu0gk9aggv79597xwazyfzaggv2pze9z7cq3p9p72tkkux9a7xaqufa792/BVT",
+      "name": "Peelworks Factory",
+      "display": "BVT0",
+      "symbol": "BVT0",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/BVT0.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/BVT0.png",
+          "theme": {
+            "primary_color_hex": "#ffef45"
```

Comment:
these colors don't seem to be representative.

Reply:
this is is the main theme color for the protocol. is this is acceptable, or is there guidance on how to determine what is representative?

Reply:
The color is only meant to represent the image it's placed in--nothing to do with 'official colors' of the project.